src/services/anchor/index.ts

```diff
@@ -194,10 +218,23 @@ export async function sep24First(sessionParams: IAnchorSessionParams): Promise<I
   const { sep24Url } = tomlValues;
 
   // at this stage, assetCode should be defined, if the config is consistent.
-  const sep24Params = new URLSearchParams({
-    asset_code: sessionParams.tokenConfig.stellarAsset.code.string,
-    amount: sessionParams.offrampAmount,
-  });
+  let sep24Params;
+
+  if (sessionParams.tokenConfig.stellarAsset.code.string === 'ARS\0') {
```

Comment:
Let's also rather use a flag in the token config instead of a hard coded condition here.